src/locales/ge/roles.json

```diff
@@ -0,0 +1,214 @@
+        "adjudicateDisputes": "Mögliche Streitfälle, die sich aus Meldungen von Nutzern ergeben, schlichten",
+        "updateContentInformation": "Informationen über Inhalte aktualisieren, damit sie korrekt sind",
+        "collaborateWithBuilders": "Arbeite mit den <0> Buildern</0> zusammen, um sowohl die Werkzeuge als auch die Benutzererfahrung zu verbessern, um die Integrität des Inhaltsverzeichnisses zu erhöhen"      },
+      "requirements": {
+        "adjudicateDisputes": "Faire Streitschlichtung und klare und transparente Kommunikation mit Interessengruppen und Teilnehmern",
+        "stake": "Besitz einer ausreichenden Menge des nativen Plattform-Tokens, um sie zu staken."
+      }
+    },
+    "contentCreator": {
+      "title": "Content Creator",
+      "overview": "Als Videoplattform sind die auf der Plattform veröffentlichten Inhalte eines der wichtigsten Assets der Plattform. Da Video ein so flexibler Medientyp ist, der eine breite Palette von Inhaltskategorien umfassen kann, steht es im Mittelpunkt. Es ist jedoch beabsichtigt, im Laufe der Zeit ein breiteres Spektrum von Inhaltstypen zu unterstützen. Die Builder von Inhalten sind diejenigen, die an der Erstellung, Veröffentlichung und Monetarisierung von Inhalten auf der Plattform beteiligt sind",
+      "responsibilities": {
+        "publishContent": "Inhalte veröffentlichen und ein Publikum aufbauen"
+      },
+      "requirements": {
+        "publishContent": "Das Recht haben, bestehende oder neue Inhalte auf der Plattform zu veröffentlichen",
+        "stake": "Besitz einer ausreichenden Menge des nativen Plattform-Tokens, um sie zu staken."
+      }
+    },
+    "contentLead": {
+      "title": "Content Lead",
+      "overview": "Der Rat hat die Befugnis, einen Verantwortlichen für die Inhaltskuratierung für das Netzwerk zu ernennen, der weitere Inhaltskuratoren einstellen kann. Der Inhaltsverantwortliche entscheidet auch über die Prioritäten für die Kuratierung. Falls erforderlich, kann der Verantwortliche nach Rücksprache mit dem Rat auch entscheiden, Kuratoren zu entlassen, die ihre Arbeit nicht angemessen ausführen.",
+      "responsibilities": {
+        "monitorPublishing": "Überwache die Veröffentlichung neuer Inhalte im Inhaltsverzeichnis und reagiere auf Berichte über beanstandete Veröffentlichungen",
+        "adjudicateDisputes": "Mögliche Streitfälle, die sich aus Meldungen von Nutzern ergeben, beurteilen",
+        "updateInformation": "Informationen über Inhalte aktualisieren, damit sie korrekt sind",
+        "manageAndCoordinate": "Verwalte und koordiniere die Aktionen der Arbeitsgruppe Inhalte"
+      },
+      "requirements": {
+        "adjudicateDisputes": "Faire Streitschlichtung und klare und transparente Kommunikation mit Interessengruppen und Teilnehmern",
+        "stake": "Besitz einer ausreichenden Menge des nativen Plattform-Tokens, um sie zu staken."
+      }
+    },
+    "membershipScreener": {
+      "title": "Membership-Screener",
```

Comment:
Registrierungs-Sichtung? Mitglieder-Aufnahme / [Auswahl](https://de.wikipedia.org/wiki/Screener_(Informatik))
[Screening](https://en.wikipedia.org/wiki/Background_check#Employment_screening) is a known term in [economics](https://en.wikipedia.org/wiki/Screening_(economics)#Labour_market) and probably needs no translation similar to [Training](https://de.wikipedia.org/wiki/Membership_Training).

Reply:
Changed to Mitgliederauswahl